src/tracer.js

```diff
@@ -524,16 +537,11 @@ module.exports.sendTraceSync = function sendTraceSync() {
  * @param {string} value value for the added label
  */
 module.exports.label = function addLabel(key, value) {
-    const tracerObj = module.exports.getTrace();
-    if (!tracerObj || !tracerObj.currRunner) {
-        utils.debugLog('Failed to label without an active tracer');
-        return;
-    }
-    let labels = {};
-    labels[key] = value;
-    labels = utils.flatten(labels);
-    Object.keys(labels).forEach((k) => {
-        eventInterface.addLabelToMetadata(tracerObj.currRunner, k, labels[k]);
+    config.setConfig({
```

Comment:
This change is not valid (we want to preserve the behavior that `addLabel` is per trace basis)